model/event/event_test.go

```diff
@@ -118,6 +118,7 @@ func (s *eventSuite) TestWithRealData() {
 		s.checkRealData(&entries[0], loc)
 		entries[0].Timestamp = entries[0].Timestamp.In(loc)
 		entries[0].ProcessedAt = entries[0].ProcessedAt.In(loc)
+		s.Equal("5949645c9acd9604fdd202d9", entries[0].ID)
```

Comment:
One last thing: replace data[idKey] at the end of this test with a (different) string, Insert it then retrieve it again,  assert it's id is as expected. Otherwise it looks like it's just js-tests so you're good to go